stylesheet.scss

```diff
@@ -3,6 +3,7 @@
   border-width: solid 1px black;
 }
 
-.overview-icon-with-label StLabel {
-  text-shadow: 0 1px 3px rgba(0, 0, 0, 0.4);
+.panel-button StLabel,
```

Comment:
This will give the top-bar icons a shadow even in the regular session state, right? Are we just relying on the fact that it's black-on-black and so invisible?